config/constants.yml

```diff
@@ -840,7 +840,6 @@ additional_detail_question_types: {
   'Text': 'text',
   'Text Area': 'textarea',
   'Radio Button': 'radio_button',
-  'Checkbox': 'checkbox',
```

Comment:
Did they decide to get rid of the checkbox field? I can give you the logic I used in the surveyor branch to get the checkboxes to be selected correctly / store data correctly. it applied to checkbox and multiple dropdown for me.